src/components/message/message.stories.mdx

```diff
@@ -1,34 +1,30 @@
 import { Story, Canvas, Meta } from '@storybook/addon-docs/blocks';
-import { withKnobs, text, boolean } from '@storybook/addon-knobs';
 import messageDemo from './demo/default-message.twig';
-import messageWithCloseDemo from './demo/message-with-close.twig';
```

Comment:
@tylersticka I removed this since I could just use the default demo and pass in the `include_close` arg